src/StreetNameRegistry.Api.Oslo/StreetName/List/StreetNameListOsloResponse.cs

```diff
@@ -34,6 +36,12 @@ public class StreetNameListOsloResponse
         [DataMember(Name = "Volgende", Order = 2, EmitDefaultValue = false)]
         [JsonProperty(Required = Required.Default, DefaultValueHandling = DefaultValueHandling.Ignore)]
         public Uri? Volgende { get; set; }
+
+        [JsonIgnore]
+        internal SortingHeader Sorting { get; set; }
```

Comment:
Why is this added with JsonIgnore?

Reply:
The controller needs this to add to the `Response` (HttpResponse).